### PR TITLE
preview - Update of identityhash code

### PR DESCRIPTION
**Pull request by Michkov**
Based on comments in #763

### CODE REVIEW

**Review by amfred — APPROVED** [review]
lgtm

**Comment by MatousJobanek** [comment]
How about the flow of entering the workspaces: https://github.com/redhat-appstudio/infra-deployments/pull/763#discussion_r982244296 shouldn't this be fixed as well?

plus which name of the application-service APIExport should we use? https://github.com/redhat-appstudio/infra-deployments/pull/763#discussion_r982274493 - is this the same as for https://github.com/redhat-appstudio/infra-deployments/pull/763#discussion_r983191555?

File: components/application-service/base/kustomization.yaml [hunk 1]
```diff
@@ -13,3 +13,12 @@ namespace: application-service
...
   - ./manager_resources_patch.yaml
+
+# APIExport managed by infra-deployments for now
+patchesStrategicMerge:
+- |-
+  apiVersion: apis.kcp.dev/v1alpha1
+  kind: APIExport
+  metadata:
+    name: has
+  $patch: delete
```
Comment (MatousJobanek) [inline components/application-service/base/kustomization.yaml:24]: Just asking, but why couldn't we delete the `has` APIExport from the application-service repo (for example as part of https://github.com/redhat-appstudio/application-service/pull/192)
I thought that our goal is to have one APIExport that would be used as the only source of truth. Or am I missing something?

Comment (Michkov) [reply components/application-service/base/kustomization.yaml]: Removed the removing part from this PR, handled in #790  for has

**Pull request merged**